test/common/run-tests

```diff
@@ -376,7 +376,7 @@ def main():
     parser.add_argument('--exclude', action="append", default=[], metavar="TestClass.testName",
                         help="Exclude test (exact match only); can be specified multiple times")
     parser.add_argument('-b', '--batches', type=int,
-                        default=os.environ.get("TEST_BATCHES", 1), help="Number of concurrent batches of nondestructive tests")
+                        default=os.environ.get("TEST_BATCHES", 2), help="Number of concurrent batches of nondestructive tests")
```

Comment:
How about defaulting to max(TEST_JOBS//2, 1) ? With the current measurements and evolution of serial vs. parallel tests that feels like DTRT.

 I.e. determine TEST_JOBS outside of `add_argument()`, and use it for --jobs and --batches.